config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'upload/' => 'upload#index', as: 'upload'
```

Comment:
Index is for listing the list of all the tracks. New is the page for creating new records. http://www.tutorialspoint.com/ruby-on-rails/rails-controllers.htm